### ISSUE TITLE
[CST][Spike] Investigate issues related to EVSS::DocumentUpload Sidekiq failures for Dependents

### ISSUE THREAD

**Issue by jerekshoe** [department-of-veterans-affairs/va.gov-team#74381]
## Background
We have been made aware of the fact that many `EVSS::DocumentUpload` jobs are making it into the Sidekiq Dead Job Queue. These jobs are related to document uploads from the CST and the mobile app. We are currently looking into potential resolutions for jobs that have failed with the below error message:
```
[{"key"=>"File Upload", "text"=>"Unique ID: [170502378XXXX] Server Name: [ManagedServerXXX] ", "severity"=>"ERROR"}]
```

## Tasks
- [x] Identify why these issues are happening
- [ ] Identify a resolution for these failures

## Acceptance Criteria
- [x] We have identified the reason these jobs are failing
- [x] We have determined a resolution for these job failures

**Comment by jerekshoe** [comment]
From talking to EVSS in slack (evss-prod channel), we have identified that these issues are caused by file uploads to EVSS that don't have a valid value in the `va_eauth_birlsfilenumber` header. We have strong reason to believe that this is related to uploads initiated by non-Veteran users (non-Veteran uses don't have a BIRLS ID / File number, so we don't have a value to pass in the header mentioned above). We are working to identify an appropriate resolution

**Comment by HeatherWidmont** [comment]
To discuss approach with the Enablement team in the Product sync later today

**Comment by jerekshoe** [comment]
#### Update (1/30)
I think that we have identified an interim solution, which is to send the SSN in the `va_eath_birlsfilenumber` in cases where the user does not have a BIRLS ID. There should be another ticket for that work

**Comment by HeatherWidmont** [comment]
@jerekshoe is the work you did for this spike equivalent to a 5 or should we adjust the story points for this issue?

**Comment by HeatherWidmont** [comment]
Jerek confirmed in slack that its a 5 (which is a conservative estimate for this)

**Comment by va-albers** [comment]
Watch Officer note - tickets related to [DD-190515](https://vagov.ddog-gov.com/monitors/190515) relate to this ticket.

**Comment by acrollet** [comment]
Hi folks, just checking in to see if this work is slated in an upcoming sprint? Thanks much!

**Comment by jerekshoe** [comment]
We are waiting on another team to implement changes on their side before we can do anything further with this

**Comment by sanjabaj2** [comment]
Do we need another ticket to perform manual intervention on past failed uploads? There were failures over the weekend.

**Comment by jerekshoe** [comment]
We’ve been including manual investigation in another ticket, though for the most part all we can really do is pass the details over to EVSS and hope they can give us more info

**Comment by sanjabaj2** [comment]
> We’ve been including manual investigation in another ticket, though for the most part all we can really do is pass the details over to EVSS and hope they can give us more info

Thank you. Maybe the best is to use slack notification event and document there that information was passed out to EVSS. That way veteran's application doesn't fall into cracks.
